db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField("Description of bonus types", max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True, null=True)
```

Comment:
Consider removing `null=True` from the `name` field in the `Guild` model if every guild should have a name. Allowing `null` might lead to guilds without names, which could be undesirable.